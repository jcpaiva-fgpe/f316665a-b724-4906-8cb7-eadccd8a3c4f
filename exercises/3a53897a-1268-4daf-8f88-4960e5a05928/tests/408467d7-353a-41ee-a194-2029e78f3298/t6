12778 20556 12780 20558
44
6 6
-4 -4
0 6
0 -6
-2 2
0 -30000
-30000 0
-10000 0
-12500 0
-2 2
10000 10000
10000 0
10000 10000
10000 0
0 18000
-276 -276
0 -52
0 100
0 2
0 2
0 -4
0 4
-1666 -1666
0 -4454
0 8
0 -6
2500 2500
300 -300
-22 -22
0 -600
0 -1400
0 100
0 50
0 -46
0 -100
10 10
0 4000
-2 2
0 100
-8 8
0 100
0 300
0 -6000
0 2
