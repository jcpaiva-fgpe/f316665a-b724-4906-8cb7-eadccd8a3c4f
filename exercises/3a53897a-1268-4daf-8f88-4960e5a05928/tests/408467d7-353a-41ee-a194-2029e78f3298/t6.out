5
Atenas 2
Barcelona 2
Londres 4
Madrid 4
Paris 2
